### DISCUSSION TITLE
Welcome to nativescript-pdf-view Discussions!

### DISCUSSION THREAD

**Discussion by madmas** [madmas/nativescript-pdf-view#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by burichdeveloper** [comment]
Hello! Is there a way to style the PDFVIEW component so it display the PDF centered? At the moment the PDF is display as verticalAlignment=top
